### PR TITLE
Add links to tutorial section

### PR DESCRIPTION
**Pull request by IgorTatarnikov**
Added the remaining links and updated the instructions.

### CODE REVIEW

**Comment by IgorTatarnikov** [comment]
@adamltyson @alessandrofelder Could you take a look and merge if good?

File: slides/installation.qmd [hunk 1]
```diff
@@ -4,8 +4,8 @@
 #### Silicon mac users
 Install brainglobe tools individually:
```
Comment (adamltyson) [inline slides/installation.qmd:5]:
```suggestion
```

Comment (adamltyson) [reply slides/installation.qmd:5]: The equivilant sentence can probably be removed in the other section too.

**Review by adamltyson — APPROVED** [review]
Thanks @IgorTatarnikov, just one small suggestion.

File: slides/installation.qmd (hunk 1 above)
Comment (IgorTatarnikov) [reply slides/installation.qmd]: Done!

**Pull request merged**